### ISSUE TITLE
Summary of testing the same commit on Travis

### ISSUE THREAD

**Issue by howdyjessie** [informatics-isi-edu/chaise#854]
Making this summary for @hongsudt for prioritization. I'll talk about this issue at our next standup.

I built the [same Chaise commit](https://github.com/informatics-isi-edu/chaise/commit/9e8c495c7026322463426617bcf18d893ae1bf14) 6 times by itself without other Travis BUILDS running concurrently. This commit was green from before.

Of the 6 builds, 3 passed and 3 failed. Of the failures:
1. SauceConnect couldn't establish a connection and Travis failed the build after a period of inactivity. No test cases were run.
<img width="1032" alt="failure 1" src="https://cloud.githubusercontent.com/assets/4585431/21124583/aa118306-c094-11e6-984d-299e8d6686b5.png">

2. Protractor timed out on a redirect after just 1076ms; maybe that timer could be increased.
<img width="1032" alt="failure 2" src="https://cloud.githubusercontent.com/assets/4585431/21124625/e047a57c-c094-11e6-8449-17ad76983b9c.png">

3. A DOM element was undefined when Protractor attempted to get that element's HTML content.
<img width="1032" alt="failure 3" src="https://cloud.githubusercontent.com/assets/4585431/21124679/32f06d7c-c095-11e6-844e-afa5a0ad5961.png">

**Comment by hongsudt** [comment]
@chiragsanghvi can you investigate these 3 cases?

**Comment by hongsudt** [comment]
@chiragsanghvi @howdyjessie should we close the issue? 

If so, @chiragsanghvi can you summarize what you have done for some of the above cases before closing the issue?

**Comment by howdyjessie** [comment]
@hongsudt Failure reasons 2 and 3 seem to be under control now after out sleeps-to-waits conversion, but reason 1 is still a question mark for me. I'm not sure if @chiragsanghvi looked into it yet? This issue could be closed in favor of creating a specific issue for reason 1.

**Comment by hongsudt** [comment]
@chiragsanghvi <https://github.com/chiragsanghvi> what's the status on
reason 1?

**Comment by chiragsanghvi** [comment]
The reason 1 is uncontrollable, as its an error from SauceLabs. The job couldn't open a tunnel as SauceLabs refused to. This happens due to concurrent execution most of the times or traffic.

The easiest solution is to restart the build.

**Comment by hongsudt** [comment]
ok.. Let's close this issue for now.. If there is a recurrence issue that happens frequently, we should open another issue..
